CMakeLists.txt

```diff
@@ -2,8 +2,10 @@ cmake_minimum_required(VERSION 3.0)
 
 project(warg-cv-bootcamp C CXX) 
 
+set(MODULES merge)
```

Comment:
This sets the modules variable, not updates it. If you want to add merge to the list, add it after core in the existing set call. Currently it's being immediately overwritten to only include core.